### PR TITLE
feat(triggers): adds plans service

### PR DESCRIPTION
**Pull request by jurajpiar**
- with limits and filters

Cache part fo the Triggers Filters feature in [RMKT-586](https://rsklabs.atlassian.net/browse/RMKT-586)

### CODE REVIEW

File: src/definitions.ts [hunk 1]
```diff
@@ -47,6 +47,7 @@ export enum ServiceAddresses {
   STORAGE_AVAILABLE_CAPACITY = '/storage/v0/availableCapacity',
   TRIGGERS_STAKES = '/triggers/v0/stakes',
   TRIGGERS_PROVIDERS = '/triggers/v0/providers',
+  TRIGGERS_PLANS = '/triggers/v0/offers',
```
Comment (nduchak) [inline src/definitions.ts]: This little bit confusing. Can it be TRIGGERS_OFFERS?

**Review by 0xartem — APPROVED** [review]
LGTM

File: src/definitions.ts (hunk 1 above)
Comment (jurajpiar) [reply src/definitions.ts]: ah of course... not sure what happened there :s

**Review by nduchak — APPROVED** [review]
Nice job 🥇

**Pull request merged**